### PR TITLE
Docker hw

### PR DESCRIPTION
**Pull request by AnyaRatusnyak**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - "${DEBUG_PORT}:${DEBUG_PORT}"
+    environment:
+      SPRING_APPLICATION_JSON: '{
+      "spring.datasource.url": "jdbc:mysql://mysqldb:${MYSQLDB_DOCKER_PORT}/${MYSQLDB_DATABASE}?serverTimezone=UTC",
+       "spring.datasource.username": "${MYSQLDB_USER}", 
+       "spring.datasource.password": "${MYSQLDB_ROOT_PASSWORD}",
+       "jwt.expiration":"${JWT_EXPIRATION}",
+       "jwt.secret":"${JWT_SECRET}"
+       }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (yevheniisynytsia) [inline docker-compose.yml]: please add empty line

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]
please add empty line and add only related to this task files

File: src/main/java/projectbookstore/dto/category/CreateCategoryRequestDto.java [hunk 2]
```diff
@@ -2,9 +2,11 @@
 
 import jakarta.validation.constraints.NotBlank;
 import jakarta.validation.constraints.NotNull;
-import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
 
-@Data
+@Getter
+@Setter
```
Comment (karma-o) [inline src/main/java/projectbookstore/dto/category/CreateCategoryRequestDto.java]: why not data? DTO classes also need `equals`, `hashcode` & `toString` methods

**Review by karma-o — APPROVED** [review]
Left a minor comment, GJ!

File: src/main/java/projectbookstore/dto/category/CreateCategoryRequestDto.java (hunk 2 above)
Comment (AnyaRatusnyak) [reply src/main/java/projectbookstore/dto/category/CreateCategoryRequestDto.java]: Sorry !! I had lots of conflicts after rebase and this happened again((

**Pull request merged**